projects/OpenWebCalendar/default.nix

```
@@ -0,0 +1,15 @@
+{
+  pkgs,
+  lib,
+  sources,
+  ...
+}@args:
+{
+  metadata.subgrants = [ "OpenWebCalendar" ];
+  nixos.modules.services.open-web-calendar = {
+    name = "open-web-calendar";
+    module = "${sources.inputs.nixpkgs}/nixos/modules/services/web-apps/open-web-calendar.nix";
+    examples = { };
+    links = { };
+  };
```

We should still keep the test, even if it's not linked to any particular example at the moment:

```suggestion
  };
  tests.open-web-calendar = pkgs.nixosTests.open-web-calendar;
```